packages/mobile/src/verify/saga.ts

```diff
@@ -291,12 +306,8 @@ export function* completeAttestation(
     `Completing code (${code.shortCode} ${codePosition}) for issuer: ${code.issuer}`
   )
 
-  // Make each concurrent completion attempt wait a sec for where they are relative to other codes
-  // to ensure `processingInputCode` has enough time to properly gate the tx. 0-index code
-  // will have 0 delay, 1-index code will have 1 sec delay, etc.
   if (shouldUseKomenci) {
-    yield delay(codePosition * 5000)
-    yield inputAttestationCodeLock.acquireAsync()
+    yield call([inputAttestationCodeLock, inputAttestationCodeLock.acquireAsync])
```

Comment:
The delay was necessary because the lock wasn't working as intended because it was not being called correctly. redux-saga can be non-intuitive at times :/

Reply:
I'll update the comment and add a TODO in the code to revisit that